src/css/profile/mobile/common/bottombar.less

```diff
@@ -25,6 +25,9 @@
         &.ui-bottom-bar-icons {
             padding: 0 10 * @px_base;
         }
+        &.ui-hidden {
+            display: none;
+        }
```

Comment:
How about moving the 'ui-hiddent' to a more common place? such as 'common.less'?
It seems to be more reusable.

Reply:
I remember that general selector is often too weak.